ui/apps/platform/src/Containers/Login/LoginPage.js

```diff
@@ -141,6 +141,9 @@ class LoginPage extends Component {
         if (authProviderResponse && authProviderResponse.error) {
             const errorKey = authProviderResponse.error.replace('_', ' ');
             const errorMsg = authProviderResponse.error_description || '';
+            // TODO is the link external or internal?
+            /* eslint-disable generic/ExternalLink-a */
+            /* eslint-disable generic/a-target-rel */
```

Comment:
I'd be curious to see if this could ever be an internal link. It seems like this would always be external and specific to the auth provider, and maybe `null` for auth providers that connect to ACS directly (e.g. basic auth).

Also unrelated, but interesting use of an IIFE here instead of just using a ternary directly.

Reply:
Aha, that is helpful insight! I will go ahead and apply the pattern here.